minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/pom.xml

```diff
@@ -40,6 +40,24 @@ limitations under the License.
             <groupId>org.apache.nifi.minifi</groupId>
             <artifactId>minifi-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-tomcat</artifactId>
```

Comment:
Was there a reason that Tomcat was chosen over some of the other alternatives?  No strong objection to Tomcat, but know that Jetty is used elsewhere.

pippo-core looks clear, but pippo-tomcat looks like it will need the LICENSE/NOTICE files updated as Tomcat seems to bring in a fair number of items